birdhouse/optional-components/wps-healthchecks/docker-compose-extra.yml

```diff
@@ -5,13 +5,13 @@ services:
       test:
         [
           "CMD",
-          "curl",
-          "--insecure",
-          "--fail",
-          "https://localhost${TWITCHER_PROTECTED_PATH}/catalog?service=WPS&request=GetCapabilities",
+          "python",
+          "-c",
+          "import requests; \
+           assert requests.get('http://localhost?service=WPS&request=GetCapabilities').status_code == 200",
```

Comment:
Was using twitcher path served on port 80, but now are you sure the catalog is available at port 80? I guess that if the healthcheck is ok it's fine.

Reply:
Yes, I updated the healthchecks by doing each of the adjusted request manually within each corresponding container. 
The command is executed from within the container, so twitcher path causes http not found.